django/contrib/admin/templates/admin/includes/fieldset.html

```diff
@@ -20,7 +20,13 @@
                             {% else %}
                                 {{ field.label_tag }}
                                 {% if field.is_readonly %}
-                                    <div class="readonly">{{ field.contents }}</div>
+                                    <p>
```

Comment:
Look like this needs to be `<div class="readonly">` for some existing tests

Reply:
true, fixed it.